app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_cat_and_dog_years() -> None:
+    assert get_human_age(32, 29) == [4, 3]
+
+
+def test_should_return_the_cat_is_older() -> None:
+    assert get_human_age(28, 28) == [3, 2]
+
+
+def test_should_return_zero_when_less_years() -> None:
+    assert get_human_age(10, 12) == [0, 0]
+
+
+def test_should_return_one_year_in_15_to_24() -> None:
+    assert get_human_age(17, 22) == [1, 1]
```

Comment:
Follow a [checklist](https://github.com/donbas777/py-cat-and-dog-years/blob/develop/checklist.md#clean-code) carefully and test all cases mentioned there including error processing using `@pytest.mark.parametrize`